test/runAll.spec.js

```diff
@@ -35,6 +43,23 @@ describe('runAll', () => {
     return expect(runAll(scripts, getConfig({}))).resolves.toEqual('No tasks to run.')
   })
 
+  it('should resolve the promise with no files', () => {
```

Comment:
Instead of `expect.anything`, let's use `console.printHistory()`. Also, since we have `babel-jest` installed, we can use `async-await` here. So change the test like so:

```js
  it('should resolve the promise with no files', async () => {
    await runAll(scripts, getConfig({ linters: { '*.js': ['echo "sample"'] } }))
    expect(console.printHistory()).toMatchSnapshot()
  })
```